etc/docs/preview.ts

```diff
@@ -0,0 +1,36 @@
+import { on } from 'events';
```

Comment:
I retract my request for this, apologies! Let's go with the python solution or we can use `npx http-server docs` which will install it as needed.

Reply:
Yes!

Reply:
Filed a ticket here - https://jira.mongodb.org/browse/NODE-4056